ManageIQ/manageiq
# Load all gems for MiqUiWorker and MiqGenericWorker

Both of these workers require almost everything in the bundle for manageiq, so load the default bundler group.  If needed, we can update these later after dependencies are sorted out, but for now, just load everything.

Links
-----

* PRs where this has been a problem:
  * https://github.com/ManageIQ/manageiq/pull/15669
  * https://github.com/ManageIQ/manageiq/pull/15818
  * https://github.com/ManageIQ/manageiq/pull/15820


Steps for Testing/QA
--------------------

You can try running the UI worker in foreman (for example) using `ruby` instead of `bundle exec rails r`, and it should allow you to log in properly.  Or you can just run either the `MiqUiWorker` or a `MiqGenericWorker` with the `run_single_worker.rb` script manually:

```console
$ ruby lib/workers/bin/run_single_worker.rb MiqUiWorker
```

Same diff
